iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Bus.java

```diff
@@ -240,4 +240,14 @@
      */
     void visitConnectedOrConnectableEquipments(TopologyVisitor visitor);
 
+    /**
+     * Get an iterable of the terminals connected to the bus.
+     */
+    Iterable<? extends Terminal> getConnectedTerminals();
```

Comment:
I think you should to provide a default behavior to not break existing code (powsybl-network-store)